contracts/lockdrop/src/contract.rs

```diff
@@ -64,10 +69,16 @@ pub fn instantiate(
         ));
     }
 
+    if msg.max_positions_per_user < MIN_POSITIONS_PER_USER {
```

Comment:
I think we can check it just using this: ```if msg.max_positions_per_user == 0```

Reply:
Let's use a constant, in case the minimum value might change.